src/main/java/FlowManager.java

```diff
@@ -0,0 +1,72 @@
+import domain.JanggiManager;
+import domain.piece.Team;
+import domain.util.ErrorHandler;
+import view.InputView;
+import view.MoveCommand;
+import view.OutputView;
+import view.SangMaOrderCommand;
+
+public class FlowManager {
+
+    private static final Team START_TEAM = Team.CHO;
+
+    public void startGame() {
+        OutputView.printStart();
+
+        JanggiManager janggiManager = createJanggiManager();
+        OutputView.printBoard(janggiManager.board());
+
+        Turn turn = new Turn(START_TEAM);
+        boolean isRunning = true;
+        while (isRunning) {
+            isRunning = movePieceByTurn(janggiManager, turn);
+        }
```

Comment:
~~장기게임 객체를 하나 만들어서 이 객체에게 게임의 흐름을 관리하는 역할을 맡겨보면 어떨까요?~~

`JanggiManager` 라는 객체에게 장기게임의 흐름을 관리하는 역할을 맡기면 될 것 같아요. 😃

```suggestion
        while (janggiGame.isRunning()) {
            janggiManager.movePieceByTurn();
        }
```

Reply:
로키 말씀대로 구 JanggiManager(현 Board)가 왕이 살아있는지 여부를 알고 있어서 그게 더 자연스러운 것 같아요! 
현 Board가 게임 종료 여부를 알려주도록 수정했습니다 😄 
```java
while (!board.isEnd()) {
        OutputView.printBoard(board);
        processTurn(board, turn);
        turn.changeTurn();
    }
    OutputView.printMatchResult(board.findWinTeam());
```